### PR TITLE
Basic "mc" utility with commandline parsing

### PR DESCRIPTION
**Pull request by gwicke**

### CODE REVIEW

File: bin/mc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import argparse
+import subprocess
+from subprocess import check_output, Popen, PIPE, STDOUT
+
+
+def get_minikube_ip():
+    """
+    Get the minikube ip
+    """
+    # XXX: Sudo is needed on Linux. What about OSX and Win?
```
Comment (Pchelolo) [inline bin/mc]: No need for sudo on OS X

**Comment by Pchelolo** [comment]
Perhaps it would be better to use a python client for talking to `kubectl` right away instead of shelling out?

**Comment by gwicke** [comment]
> Perhaps it would be better to use a python client for talking to kubectl right away instead of shelling out?

If that works well with normal installs (picks up configs, keys etc), then sure. It's a bit out of scope for this PR, though.

**Comment by Pchelolo** [comment]
+1 overall

File: bin/mc (hunk 1 above)
Comment (d00rman) [reply bin/mc]: Neither do I need it on my install. This should definitely be a configuration option residing in `$HOME/.mwc-rc` or similar.

**Comment by d00rman** [comment]
Modulo the sudo case (which we can address in a separate PR), :+1: 

However, have we not said that we shouldn't call the utility `mc` as that is the command for the Midnight Commander? `mwc` anyone?

File: bin/mc (hunk 1 above)
Comment (gwicke) [reply bin/mc]: Auto-detecting this would be even nicer for users.

**Comment by gwicke** [comment]
> mwc anyone?

Another option: `mwctl`

**Comment by d00rman** [comment]
> Another option: `mwctl`

I like the direction, but reads like _mediawiki control_ to me, which is not really what it is. I don't have a better proposition though. @Pchelolo @eevans ? :P

**Comment by Pchelolo** [comment]
`mwctl` sounds fine to me

**Comment by gwicke** [comment]
> but reads like mediawiki control to me, which is not really what it is.

If we call the overall system "MediaWiki" of some sort (as in mediawiki-containers), then this kind of makes sense. It's the only name we have so far for a service-based wiki system offering the Wikipedia experience.

**Comment by d00rman** [comment]
Alright. So rename the file and let's merge this baby

**Pull request merged**